pkg/reconciler/ingress/controller.go

```diff
@@ -179,3 +182,10 @@ func combineFunc(functions ...func(interface{})) func(interface{}) {
 		}
 	}
 }
+
+func getSecretInformer(ctx context.Context) v1.SecretInformer {
+	if pkg.ShouldFilterByCertificateUID() {
+		return secretfilteredinformer.Get(ctx, networking.CertificateUIDLabelKey)
+	}
+	return secretfilteredinformer.Get(ctx, "") // Allow all
```

Comment:
use non-filtered informer

Reply:
You cant do that because when you import the related file it will initialize the informer and that informer will suffer from the memory leak even when you do use a filtered informer. I have tested that.

Reply:
Example of this kind of failure, logs and patch [here](https://gist.github.com/skonto/7f0a93e376546eb957ae2f3687e8c062).